### ISSUE TITLE
Add missing tests for polling commands

### ISSUE THREAD

**Issue by dmoranj** [telefonicaid/iotagent-ul#170]
Multiple unit tests where defined in the Mocha script, but just one of them was implemented. This gap should be filled.

**Comment by Keshav-NEC** [comment]
Hi @fgalan  @AlvaroVega , does this issue still exist. If yes, I would like to contribute on this.

**Comment by fgalan** [comment]
> Hi @fgalan @AlvaroVega , does this issue still exist. If yes, I would like to contribute on this.

Yes, this issue is still valid. It would great if you could contribute to increase the test base adding new ones covering the polling command functionality (in this repo and also in https://github.com/telefonicaid/iotagent-json... in fact, that last one is more prioritary for us)

**Comment by Keshav-NEC** [comment]
Hi @fgalan , I have searched for the `API` regarding polling commands in the document but can't get the `API`. Can you guide me for the document section of polling commands `API`.
